src/components/IframeReport.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { getContextPath } from '../utils/api'
+
+const LOADING_DONE = 'DHIS2_REPORTS_APP_IFRAME_CONTENT_LOADING_DONE'
+const ADDITIONAL_STYLE_RULES = `
+body {
+    font-family: 'Roboto', sans-serif;
+}
+table {
+    border-collapse: collapse;
+}
+th, td {
+    border: 1px solid #bcbcbc;
+    padding: 4px;
+}
+`
+
+class IframeReport extends React.PureComponent {
+    constructor(props) {
```

Comment:
With the 4 changes I recommend for the constructor body, the construcor method can be removed.